packages/protocol/contracts/tokenvault/ERC721Vault.sol

```diff
@@ -145,11 +145,14 @@ contract ERC721Vault is BaseNFTVault, IERC721Receiver {
         }
 
         (
+            ,
             CanonicalNFT memory nft, //
             ,
             ,
             uint256[] memory tokenIds
-        ) = decodeMessageData(message.data);
+        ) = abi.decode(
```

Comment:
@adaki2004 this is the change I want, but I will lose connection soon, so please continue this change (remove all decodeMessageData)